src/server/routes/index.js

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+import map from './map';
+
+const router = express.Router();
+router.use('/map', map);
+
+export default router;
```

Comment:
Yep. we need to keep our style the best. That is a posix standard: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206